plugins/sql.c

```diff
@@ -334,6 +334,26 @@ static int sqlite_authorize(void *dbq_, int code,
 			return SQLITE_OK;
 		if (streq(b, "total"))
 			return SQLITE_OK;
+		if (streq(b, "date"))
+			return SQLITE_OK;
+		if (streq(b, "datetime"))
+			return SQLITE_OK;
+		if (streq(b, "julianday"))
+			return SQLITE_OK;
+		if (streq(b, "strftime"))
+			return SQLITE_OK;
+		if (streq(b, "time"))
+			return SQLITE_OK;
+		if (streq(b, "timediff"))
+			return SQLITE_OK;
+		if (streq(b, "unixepoch"))
+			return SQLITE_OK;
+		if (streq(b, "current_date"))
```

Comment:
`current_date` `current_time` and `current_timestamp` are [keywords for SQLite](https://www.sqlite.org/lang_keywords.html), not functions. as far as i can tell via the SQLite documentation, they're only applicable/useful for setting the default value in a CREATE TABLE statement which iiuc isn't useful for running queries. see the default value [section 3.2](https://www.sqlite.org/lang_createtable.html)

Would recommend removing them from this commit.

Reply:
Yes you are right, I realised I can use the 'now' argument to the datetime functions to get the current date and time.